### DISCUSSION TITLE
关于公告题的题库

### DISCUSSION THREAD

**Discussion by Adoretheall007** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#33] [Q&A] [answered]
希望新增一个关于答题注意事项的题库答案整合

**Comment by Adoretheall007** [comment]
![QQ截图20240109173853](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/154971368/165806b9-6109-417a-8154-6ac3c18e40a8)

**Comment by xbn2002** [comment, marked as answer]
现在也没有【答题注意事项】字样了，有时候公告类型题也会挖个小坑，比如“发烧友……，但会肆意谩骂”，不过这种坑仔细点看能避免。
另外公告类型题也一直就那么几道，几乎周周不变。每周换题后，敬请关注解析。
